src/framework/card/_card.scss

```diff
@@ -35,15 +35,36 @@
     }
 
     &__footer {
+      display: flex;
+      flex-direction: row;
+      height: 24px;
+      justify-content: center;
+      text-align: center;
```

Comment:
I think those styles are a little overkill. `display: flex` could be fine and the text-alignment should only be applied to the dropdown content (or used somewhere else?). Also the specified heights here and in status doesn't seem to be needed.